### ISSUE TITLE
Problems running playground

### ISSUE THREAD

**Issue by patoroco** [LoganWright/Genome#50]
Hi,

I'm having some troubles trying to run playground. I need to drag `Genome.xcodeproj` inside of _Genome.playground > Sources_ folder because if I don't do it, it has problems finding `MappableObject` and other requirements in playground.

Do you know if I'm doing something wrong? I usually don't use playgrounds, so maybe it's a problem of mine...

**Comment by LoganWright** [comment]
@patoroco To use the playground there are a few steps:

1. Open `Genome.xcworkspace` -- Make sure its the workspace, not the project or playground
2. Select and build Genome-iOS from the targets
3. Now use Playground inside of the workspace

Let me know if that still doesn't work for you for some reason.

**Comment by patoroco** [comment]
Oh, what strange!

I just clone from git repo now, do `pod install`, and after that open `.xcworkspace` file as is usual while working with CocoaPods, and after build Genome-iOS as you told me, it runs correctly.

Yesterday I did that in the same way (I think that :dancers:), and it returned me several errors which I solved drag-drop `.xcodeproj` inside of Genome.

Anyway, it's running correctly, so I close this issue. Thanks for your support @LoganWright
